src/app/core/services/netsuite/netsuite-core/netsuite-connector.service.ts

```
@@ -36,18 +62,77 @@ export class NetsuiteConnectorService {
+      }),
+      catchError((error) => {
+        if (error.error.message !== "Netsuite credentials not found" && shouldShowTokenExpiredMessage){
+          this.toastService.displayToastMessage(ToastSeverity.ERROR, this.translocoService.translate('netsuiteConnector.connectionExpiredToast'), 6000);
+        }
+        return of(false);
+      })
+    );
+  }
+
+  @Cacheable()
+  checkNetsuiteTokenHealth(workspaceId: string): Observable<{}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)